docs/technical_reference/property_models/modified_ADM1.rst

```diff
@@ -0,0 +1,341 @@
+   "Yield of valerate on polyhydroxyalkanoates, f_va_PHA", ":math:`f_{va,PHA}`", "f_va_PHA", 0.1, ":math:`\text{dimensionless}`"
+   "Saturation coefficient for acetate, K_A", ":math:`K_{A}`", "K_A", 4e-3, ":math:`\text{kg/}\text{m}^3`"
+   "Saturation coefficient for polyphosphate, K_PP", ":math:`K_{PP}`", "k_PP", 0.32e-3, ":math:`\text{dimensionless}`"
+   "Rate constant for storage of polyhydroxyalkanoates, q_PHA", ":math:`q_{PHA}`", "q_PHA", 3, ":math:`\text{d}^{-1}`"
+   "Yield of biomass on phosphate (kmol P/kg COD), Y_PO4", ":math:`Y_{PO4}`", "Y_PO4", 12.903e-3, ":math:`\text{dimensionless}`"
+   "Potassium coefficient for polyphosphates, K_PP", ":math:`K_{PP}`", "K_PP", 1/3, ":math:`\text{dimensionless}`"
+   "Magnesium coefficient for polyphosphates, Mg_PP", ":math:`Mg_{PP}`", "Mg_PP", 1/3, ":math:`\text{dimensionless}`"
+   "Carbon dioxide acid-base equilibrium constant, K_a_co2", ":math:`K_{a,co2}`", "K_a_co2", 4.94e-7, ":math:`\text{kmol/}\text{m}^3`"
+   "Inorganic nitrogen acid-base equilibrium constant, K_a_IN", ":math:`K_{a,IN}`", "K_a_IN", 1.11e-9, ":math:`\text{kmol/}\text{m}^3`"
+
+Properties
+----------
+.. csv-table::
+  :header: "Description", "Symbol", "Variable", "Index", "Units"
+
+  "Fluid specific heat capacity", ":math:`c_p`", "cp", "None", ":math:`\text{J/kg/K}`"
+  "Mass density", ":math:`\rho`", "dens_mass", "[p]", ":math:`\text{kg/}\text{m}^3`"
+
+Process Rate Equations
+----------------------
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Hydrolysis of carbohydrates", ":math:`\rho_1 = k_{hyd,ch} C_{X_{ch}}`"
+   "Hydrolysis of proteins", ":math:`\rho_2 = k_{hyd,pr} C_{X_{pr}}`"
+   "Hydrolysis of lipids", ":math:`\rho_3 = k_{hyd,li} C_{X_{li}}`"
+   "Uptake of sugars", ":math:`\rho_4 = k_{m_{su}} \frac{C_{S_{su}}}{K_{S_{su}}+C_{S_{su}}} C_{X_{su}} \frac{1}{1 + K_{S_{IN}}/C_{S_{IN}}/14} \cdot \frac{1}{1 + K_{S_{IP}}/C_{S_{IP}}/31} I_{pH,aa}`"
+   "Uptake of amino acids", ":math:`\rho_5 = k_{m_{aa}} \frac{C_{S_{aa}}}{K_{S_{aa}}+C_{S_{aa}}} C_{X_{aa}} \frac{1}{1 + K_{S_{IN}}/C_{S_{IN}}/14} \cdot \frac{1}{1 + K_{S_{IP}}/C_{S_{IP}}/31} I_{pH,aa}`"
+   "Uptake of long chain fatty acids (LCFAs)", ":math:`\rho_6 = k_{m_{fa}} \frac{C_{S_{fa}}}{K_{S_{fa}}+C_{S_{fa}}} C_{X_{fa}} \frac{1}{1 + K_{S_{IN}}/C_{S_{IN}}/14} \cdot \frac{1}{1 + C_{S_{h2}}/K_{I,h2_{fa}}} \cdot \frac{1}{1 + K_{S_{IP}}/C_{S_{IP}}/31} I_{pH,aa}`"
+   "Uptake of valerate", ":math:`\rho_7 = k_{m_{c4}} \frac{C_{S_{va}}}{K_{S_{c4}}+C_{S_{va}}} C_{X_{c4}} \frac{C_{S_{va}}}{C_{S_{bu}} + C_{S_{va}}} \cdot \frac{1}{1 + K_{S_{IN}}/C_{S_{IN}}/14} \cdot \frac{1}{1 + C_{S_{h2}}/K_{I,h2_{c4}}} \cdot \frac{1}{1 + K_{S_{IP}}/C_{S_{IP}}/31} I_{pH,aa}`"
+   "Uptake of butyrate", ":math:`\rho_8 = k_{m_{c4}} \frac{C_{S_{bu}}}{K_{S_{c4}}+C_{S_{bu}}} C_{X_{c4}} \frac{C_{S_{bu}}}{C_{S_{bu}} + C_{S_{va}}} \cdot \frac{1}{1 + K_{S_{IN}}/C_{S_{IN}}/14} \cdot \frac{1}{1 + C_{S_{h2}}/K_{I,h2_{c4}}} \cdot \frac{1}{1 + K_{S_{IP}}/C_{S_{IP}}/31} I_{pH,aa}`"
```

Comment:
Last I checked, the uptake equations were not rendering properly.

Reply:
I believed I've fixed these, but I am just waiting to push a bunch of changes at once

Reply:
These are good on my side now. If it's still not rendering for you, you might need to open a new browser tab and pull up this PR again (not sure why this happens but opening and closing Chrome is the only way my readthedocs will update)